openedx/openedx-platform
# [FC-0009] Remove unneeded filter from static asset copy/paste code

## Description

This is a follow-up PR to #32346. I'm just removing the added "filter" because:
1. We don't have a good use case for it yet
2. Doing so simplifies the code

## Supporting information

Ticket: https://github.com/openedx/modular-learning/issues/70

See [this discussion on the previous PR](https://github.com/openedx/edx-platform/pull/32346#discussion_r1242272529).

## Testing instructions

Copy an XBlock that uses static assets into a course and confirm the static assets are copied along with it.

## Deadline

None

## Other information

Private ref MNG-3802
